tests/distributions/test_gaussian_model.py

```diff
@@ -129,28 +129,33 @@ def test_fit():
     upper = random_number_generator.normal(loc=mean_upper, scale=sd_upper, size=n)
     weights = random_number_generator.randint(low=1, high=100, size=n)
     alpha = 0.9
-    beta = 1
     estimand = "turnout"
     model_settings = {
         "election_id": "2017-11-07_VA_G",
         "office": "G",
         "geographic_unit_type": "county",
         "save_conformalization": False,
+        "beta": 1,
+        "winsorize": True,
```

Comment:
I think this should be False now, since we have a separate `test_winsorize` unit test?